README.md

```diff
@@ -6,3 +6,8 @@ Simple Minecraft Server Install Script (Linux)
 - Give yourself permission with `chmod +x minecraftsi.sh`
 - Run it with `./minecraftsi.sh`
 - You might have to install the `dialog` package to use.
+
+or
```

Comment:
This could be changed to something along the lines of "Or alternatively (only on bash)"